location/views.py

```diff
@@ -1,7 +1,21 @@
 from django.views import generic
 from .models import Location
+from django.shortcuts import get_object_or_404
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from django.contrib.auth.decorators import login_required
+from django.contrib import messages
 
 
 class LocationView(generic.DetailView):
     model = Location
     template_name = "location.html"
+
+
+@login_required
+def favorites(request, pk):
+    apartment = get_object_or_404(Location, pk=pk)
```

Comment:
Can we have a basic test that checks http response status code 200?

Check 200:
https://github.com/gcivil-nyu-org/fall2019-cs-gy-6063-team-five_fries/blob/develop/mainapp/tests.py#L38

Please refer to the following code for testing a functionality that only works for logged in user.

Force login:
https://github.com/gcivil-nyu-org/fall2019-cs-gy-6063-team-five_fries/blob/develop/mainapp/tests.py#L15

Reply:
Thank you for the links! Added tests for `mainapp` and `location` apps.